### PR TITLE
devdoc: add bumping instructions

### PR DESCRIPTION
**Pull request by eg-ayoub**

### CODE REVIEW

File: BUMPING.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+set `$VERSION` with the appropriate value.
+
+run
+
+```
+curl https://raw.githubusercontent.com/kubernetes/ingress-nginx/helm-chart-$VERSION/deploy/grafana/dashboards/request-handling-performance.json \
+  -Lo salt/metalk8s/addons/nginx-ingress/deployed/files/ingress-nginx-performance.json
+curl https://raw.githubusercontent.com/kubernetes/ingress-nginx/helm-chart-$VERSION/deploy/grafana/dashboards/request-handling-performance.json \
+  -Lo salt/metalk8s/addons/nginx-ingress/deployed/files/ingress-nginx-performance.json
```
Comment (eg-ayoub) [inline BUMPING.md:87]: same command twice ?

File: BUMPING.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+## Images
+
+A few tips to bump image versions and SHAs:
+
+ - bumps are done in the file `buildchain/buildchain/versions.py`.
+ - the registry for an image can be found by parsing `constants.py` and `image.py`.
+ - when the registry is known, the SHA for the new version can be fetched:
+   ```gcrane digest $registry/$image:$tag```
```
Comment (eg-ayoub) [inline BUMPING.md:127]: add skopeo command too ? or make sure it's in the devcontainer

File: BUMPING.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+## Operator-sdk and Go version
```
Comment (eg-ayoub) [inline BUMPING.md:129]: for operator and storage-operator

File: BUMPING.md (hunk 2 above)
Comment (eg-ayoub) [reply BUMPING.md:127]: maybe crane instead

**Review by TeddyAndrieux — APPROVED** [review]

**Pull request merged**